<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemanticEditor</class>
 <widget class="QDialog" name="SemanticEditor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>624</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RuleEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="gBConnections">
     <property name="title">
      <string>Semantic objects Editor</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="listViewObjects"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAddObject">
            <property name="text">
             <string>Add Object</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveObject">
            <property name="text">
             <string>Remove Object</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableViewInput"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAddInput">
            <property name="text">
             <string>Add Input</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveInput">
            <property name="text">
             <string>Remove Input</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableViewOutput"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pbAddOutput">
            <property name="text">
             <string>Add Output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbRemoveOutput">
            <property name="text">
             <string>Remove Output</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
